### PR TITLE
nahraway image detection

### LINKED ISSUE #19: write nahraway section

### PR DESCRIPTION
**Pull request by LeviUniAcc**
@1Maxnet1 @EhrenbruderRoschtbraten

### CODE REVIEW

File: paper.tex
Comment (EhrenbruderRoschtbraten) [inline paper.tex]: Maybe write a little bit more about swin-tiny-patch4-window7-224

Comment (RaphaW) [reply paper.tex]: Or maybe after merging the resnet18 detector write something like "similiar to the aforementioned detector, this one is also fintuned on the AI or not dataset .... "

Because if we don't do it like that the whole text in itself will sound very choppy

**Review by RaphaW — COMMENTED** [review]
I would wait till the other request is merged and than rewrite this one

File: paper.tex
Comment (LeviUniAcc) [reply paper.tex]: yea idk i would check for fluent text reading after we merged everything and not before couse i think we have to change it later anyway again.

Comment (1Maxnet1) [inline paper.tex]: Here is a suggested addition. Additionally we need to fix the citation here.
```suggestion
The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 finetuned on the AI or Not dataset. Swin-Transformer by Microsoft is a new Vision Transformer which makes use of hierarchical windows. This reduces the self-attention to not-overlapping windows of other areas of the IMAGE. \cite{https://arxiv.org/pdf/2103.14030.pdf}.
```
I would not wait on the other PR, but rather merge them sooner, then later to get a full text soon, which we then can rephrase if necessary.

Comment (RaphaW) [reply paper.tex]: Yes but this garantees that we do at least twice the work, otherwise if we do it halfway decent the first time, we could prevent the possibility of doing it twice
But I'll get your point

Comment (RaphaW) [reply paper.tex]: Don't like the two 'Transfomer' in one sentence so close together

**Review by RaphaW — APPROVED** [review]
If you change the bits of max, I'm fine with that

**Comment by RaphaW** [comment]
As well as the citation here as well sorry

File: paper.tex
Comment (LeviUniAcc) [reply paper.tex]: its 2 names, cant romove one transformer/replace no?

**Review by 1Maxnet1 — APPROVED** [review]
LGTM

**Pull request merged**
